tests/test_unit/approval_client/config.py

```diff
@@ -762,6 +763,22 @@
             ],
         },
     ),
+    APIResource(
+        api_name="agents",
+        resource_cls=Agent,
+        list_cls=AgentList,
+        _write_cls=AgentUpsert,
+        _write_list_cls=AgentUpsertList,
+        methods={
+            "create": [Method(api_class_method="upsert", mock_class_method="create_multiple")],
+            "retrieve": [
+                Method(api_class_method="retrieve", mock_class_method="return_values"),
+            ],
+            "delete": [
+                Method(api_class_method="delete", mock_class_method="delete_id_external_id"),
+            ],
+        },
+    ),
```

Comment:
This is updating the ApprovalClient we use in Toolkit for testing. Essentially simulating the ToolkitClient so we can intercept all CDF calls. For more information https://github.com/cognitedata/toolkit/tree/main/tests#approvalcogniteclient